app/src/main/java/woowacourse/movie/repository/db/ReservationTicketEntity.kt

```diff
@@ -0,0 +1,27 @@
+package woowacourse.movie.repository.db
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import woowacourse.movie.domain.model.reservation.seat.SelectedSeats
+import woowacourse.movie.presentation.uimodel.TicketUiModel
+
+@Entity(tableName = "reservationTicket")
+data class ReservationTicketEntity(
+    @PrimaryKey(autoGenerate = true)
+    val ticketId: Long = 0,
+    val movieTitle: String,
+    val theaterName: String,
+    val screenDate: String,
+    val screenTime: String,
+    val seats: SelectedSeats,
+)
+
+fun ReservationTicketEntity.toTicketUiModel(): TicketUiModel {
+    return TicketUiModel(
+        ticketId = this.ticketId,
+        title = this.movieTitle,
+        startTime = this.screenTime,
+        screeningDate = this.screenDate,
+        theaterName = this.theaterName,
+    )
+}
```

Comment:
```suggestion
data class ReservationTicketEntity(
    @PrimaryKey(autoGenerate = true)
    val ticketId: Long = 0,
    val movieTitle: String,
    val theaterName: String,
    val screenDate: String,
    val screenTime: String,
    val seats: SelectedSeats,
) {
    fun toTicketUiModel(): TicketUiModel = TicketUiModel(
        ticketId = this.ticketId,
        title = this.movieTitle,
        startTime = this.screenTime,
        screeningDate = this.screenDate,
        theaterName = this.theaterName,
    )
}
```

굳이 ReservationTicketEntity 바깥에 `ReservationTicketEntity.toTicketUiModel()` 을 선언하신 이유가 있나요?

Reply:
> 굳이 ReservationTicketEntity 바깥에 `ReservationTicketEntity.toTicketUiModel()` 을 선언하신 이유가 있나요?

확장함수를 사용해본 경험이 별로 없어서 사용해봤습니다 .. :sweat_smile: